website/docs/mutation.md

```diff
@@ -200,10 +200,11 @@ spec:
   parameters:
     operation: prune
```

Comment:
can we also update either the operation or the name to make sure they are consistently either adding or removing?

Reply:
Done